Source/g_game.c

```diff
@@ -660,6 +667,31 @@ static void G_DoLoadLevel(void)
     }
 }
 
+extern int ddt_cheating;
+
+static void G_ReloadLevel(void)
+{
+  int i;
+
+  if (demorecording)
```

Comment:
It should still restart from `startmap`, just not when recording a multiplayer demo.

Reply:
Oh, and `startepisode` as well. You know, UMAPINFO and stuff. :wink:

Reply:
There is no way to know if one of the players is recording a demo in netgame. Alternatively, we can always restart from startmap in netgame.